From 5152761e4bb32d0a11dcfa2b064fd10a0e5e3163 Mon Sep 17 00:00:00 2001
Message-Id: <patch-v4-5.7-5152761e4bb-20210730T174308Z-avarab@gmail.com>
In-Reply-To: <cover-v4-0.7-00000000000-20210730T174308Z-avarab@gmail.com>
References: <cover-v3-0.7-00000000000-20210730T092843Z-avarab@gmail.com>
	<cover-v4-0.7-00000000000-20210730T174308Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Wed, 21 Jul 2021 23:36:28 +0200
Subject: [PATCH v4 5/7] http: drop support for curl < 7.19.3
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

In the last commit we dropped support for curl < 7.18.0, let's
continue that and drop support for versions older than 7.19.3. This
allows us to simplify the code by getting rid of the "#ifdef" for
CURLAUTH_DIGEST_IE, that flag was added in 7.19.3[1].

1. https://curl.se/libcurl/c/CURLOPT_HTTPAUTH.html

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 http.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/http.c b/http.c
index 15b43483e5f..7b1e44fe10c 100644
--- a/http.c
+++ b/http.c
@@ -122,9 +122,7 @@ static int http_auth_methods_restricted;
 /* Modes for which empty_auth cannot actually help us. */
 static unsigned long empty_auth_useless =
 	CURLAUTH_BASIC
-#ifdef CURLAUTH_DIGEST_IE
 	| CURLAUTH_DIGEST_IE
-#endif
 	| CURLAUTH_DIGEST;
 
 static struct curl_slist *pragma_header;
-- 
2.32.0.1071.g36f34456314

